kubejs/server_scripts/create_hypertube/recipes.js

```diff
@@ -0,0 +1,15 @@
+function registerCreateHypertubeRecipes(event) {
+	event.remove({ mod: 'create_hypertube' })
+
+	event.recipes.gtceu.assembler('create_hypertube:hypertube')
+		.itemInputs('2x #forge:plates/stainless_steel', '4x ae2:quartz_glass')
+		.itemOutputs('8x create_hypertube:hypertube')
+		.EUt(512)
+		.duration(40)
+	event.recipes.gtceu.assembler('create_hypertube:entrance')
```

Comment:
could you add a recipe for the other lategame rubber? PBI or whatever it's called, just for standardization with how other gt stuff works